psi4/src/psi4/libmints/twobody.h

```diff
@@ -316,6 +321,9 @@ class PSI_API TwoBodyAOInt {
 
     /// Results go back to buffer_
     void pure_transform(int, int, int, int, int nchunk, bool copy_to_source = true);
+
+    /// Manually set up sieve if desired
+    virtual void initialize_sieve();
```

Comment:
There's already a `setup_sieve` how is this function different and how is the user supposed to know which to use?